src/diffusers/loaders.py

```diff
@@ -2591,7 +2465,7 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
         # pipeline.
 
         # Remove any existing hooks.
-        if is_accelerate_available() and is_accelerate_version(">=", "0.17.0.dev0"):
+        if is_accelerate_available() and version.parse(importlib.metadata.version("accelerate")) >= version.parse("0.17.0"):
```

Comment:
For some reason this test was always failing on my end with the previous logic, I replaced it with that one which is a one-liner (maybe there was a silent bug in the `is_accelerate_version` method